openmetadata-ui/src/main/resources/ui/src/locale/languages/de-de.json

```diff
@@ -605,6 +605,7 @@
     "in-lowercase": "in",
     "in-open-metadata": "In OpenMetadata",
     "in-review": "In Review",
+    "inactive": "Inactive",
```

Comment:
Please translate and update it with help of AI

Reply:
done